content/docs/1.8.0/nodes-and-volumes/volumes/expansion.md

```diff
@@ -125,20 +125,26 @@ If you cannot enable it but still prefer to do online expansion, you can:
 
 #### RWX volume
 
-Longhorn currently does not support fully automatic expansion of the filesystem (NFS) for RWX volumes.  You can expand the filesystem manually using one of the following methods:
+From v1.8.0, Longhorn supports fully automatic online expansion of the filesystem (NFS) for RWX volumes.  The feature requires the v1.8.0 versions of these components to be running:
 
-##### Online
-1. Expand the block device of the RWX volume via PVC or UI.
-2. Identify the Share Manager pod of the RWX volume (typically named `share-manager-<volume name>`), and then run the filesystem expansion command in it.
-    ```shell
-    kubectl -n longhorn-system exec -it <the share manager pod> -- resize2fs /dev/longhorn/<volume name>
-    ```
+- Longhorn-Manager
+- CSI plugin
+- Share Manager, which manages the NFS export
 
-> **Important**:  
-> Online expansion is possible only for `ext4` volumes. Attempts to manually expand `xfs` volumes with `xfs_growfs` may initially appear to be successful, but issues occur when the workload is scaled up and the volume is reattached. In particular, the pods become stuck in the `ContainerCreating` state, and the logs show an error message about attempts to mount the filesystem.
+If you have upgraded from a previous version, the Share Manager pods (one for each RWX volume) are not upgraded automatically, to avoid disruption during the upgrade. 
+
+After growing the block device, the CSI layer sends a resize command to the Share Manager to grow the filesystem within the block device.  With a down-rev share-manager, the command fails with an "unimplemented" error code and so no expansion happens.  To get the right image before the expansion, the simplest thing is to force a restart of the pod.  Identify the Share Manager pod of the RWX volume (typically named `share-manager-<volume name>`) and delete it:  
+
+```shell
+kubectl -n longhorn-system delete pod <the share manager pod>
+```
```

Comment:
I know this replies on RWX volume's failover feature. Can we provide two methods
1. Scale down the workloads using the RWX volume
2. Delete the share-manager pod and recreate it. Need to mention why this works.

WDYT?

Reply:
I think the scale-down is covered by the `Offline` section below.  
Are you asking that we make it clearer that these are alternatives by numbering them 1 and 2?